src/main/kotlin/no/nav/syfo/personstatus/domain/PersonOversiktStatus.kt

```diff
@@ -1,16 +1,51 @@
 package no.nav.syfo.personstatus.domain
 
+import no.nav.syfo.personstatus.api.v2.PersonOversiktStatusDTO
+import no.nav.syfo.personstatus.api.v2.toOppfolgingstilfelleDTO
+
 data class PersonOversiktStatus(
     val veilederIdent: String?,
     val fnr: String,
-    val navn: String,
+    val navn: String?,
     val enhet: String,
     val motebehovUbehandlet: Boolean?,
     val moteplanleggerUbehandlet: Boolean?,
     val oppfolgingsplanLPSBistandUbehandlet: Boolean?,
     val oppfolgingstilfeller: List<Oppfolgingstilfelle>
 )
 
+fun List<PersonOversiktStatus>.addPersonName(
+    maybePersonIdentNameMap: Map<String, String>,
+): List<PersonOversiktStatus> {
+    if (maybePersonIdentNameMap.isEmpty()) {
+        return this
+    }
+    return this.map { personOversiktStatus ->
+        val personIdent = personOversiktStatus.fnr
+        if (personOversiktStatus.navn.isNullOrEmpty()) {
+            personOversiktStatus.copy(
+                navn = maybePersonIdentNameMap[personIdent]
+            )
+        } else {
+            personOversiktStatus
+        }
```

Comment:
Denne vil vel, sånn koden er nå, alltid bli true, siden vi kun henter navn på de som mangler? Det kan jo også muligens være "riktig" å bare overskrive navnet som finnes uansett, hvis vi har fått inn et nytt?

Reply:
Nei, faktisk ikke. Den blir false dersom et element i listen er en personOversiktStatus som er opprettet i databasen før denne koden går i prod, mens den blir true dersom det er en person som er lagret i databasen etter at denne koden går i prod.

Så List<PersonOversiktStatus> (this) kan inneholde både PersonOversiktStatus som ikke har navn og som har navn.